.github/workflows/release-deploy-all.yml

```diff
@@ -72,6 +72,7 @@ env:
   HOST_PREFIX: ${{ secrets.HOST_PREFIX }}
   BUILD_NAMESPACE: ${{ vars.BUILD_NAMESPACE }}
   KEYCLOAK_REALM: ${{ vars.KEYCLOAK_REALM }}
+  IS_FULLTIME_ALLOWED: ${{ vars.IS_FULLTIME_ALLOWED }}
```

Comment:
During our conversation yesterday, I forgot to mention about `sources` as well. We need to have this environment variable going to docker compose and good to have in sources/make as well